packages/webpack-cli/lib/groups/ZeroConfigGroup.js

```diff
@@ -32,17 +34,19 @@ class ZeroConfigGroup extends GroupHelper {
                     return NONE;
                 }
             }
-            if (this.args.noMode && this.args.mode) {
-                logger.warn('You Provided both mode and no-mode arguments. You Should Provide just one. "mode" will be used.');
-            }
+
             if (this.args.mode) {
+                if (this.args.mode !== PRODUCTION && this.args.mode !== DEVELOPMENT && this.args.mode !== NONE) {
+                    logger.warn('You provided an invalid value for "mode" option.');
```

Comment:
`You provided an invalid value for "mode" option. Using "production" by default` (maybe better)

Reply:
done :+1: